readme.md

```diff
@@ -56,28 +56,51 @@ apps. Pass an empty string to disable it.
 
 Directory for data files.
 
+Example locations (with the default `nodejs` [suffix](#suffix)):
+
+- macOS: `~/Library/Application Support/MyApp-nodejs`
+- Windows: `%LOCALAPPDATA%\MyApp-nodejs\Data` (e.g., `C:\Users\USERNAME\AppData\Local\MyApp-nodejs\Data`)
```

Comment:
```suggestion
- Windows: `%LOCALAPPDATA%\MyApp-nodejs\Data` (for example, `C:\Users\USERNAME\AppData\Local\MyApp-nodejs\Data`)
```

Not everyone knows `e.g.`

Reply:
Agree, improved in 4546c80173efbea42ed80c69efa26ea2e6818183.